plugins/PrivacyManager/SystemSettings.php

```diff
@@ -26,11 +27,19 @@ class SystemSettings extends \Piwik\Settings\Plugin\SystemSettings
     /** @var Setting */
     public $showInEmbeddedWidgets;
 
+    /** @var Setting */
+    public $defaultOptedInText;
+
+    /** @var Setting */
+    public $defaultOptedOutText;
+
     protected function init()
     {
         $this->privacyPolicyUrl = $this->createPrivacyPolicyUrlSetting();
         $this->termsAndConditionUrl = $this->createTermsAndConditionUrlSetting();
         $this->showInEmbeddedWidgets = $this->createShowInEmbeddedWidgetsSetting();
+        $this->defaultOptedInText = $this->createDefaultOptedInTextSetting();
```

Comment:
How does this work for different languages? Currently, we have a feature to specify the language in the URL and we pick the correct language depending on browser setting... in this case would we need to mention it overwrites all languages? @diosmosis

Reply:
Didn't know it would be translated by language (or maybe forgot). Maybe this is best solved by the custom translations plugin?

For this feature I suppose we should say it will only be translated if the default text is used, then link to the custom translations plugin.

Reply:
Actually, I wonder if this would be solvable by just using the custom translations plugin? w/o the need for a new setting. @tsteur

Reply:
In Custom Translations there is no logic yet to translate any core translations / translation keys. Only entities and tracked values (eg dashboard names, custom dimension names, tracked event values, ...). However, https://plugins.matomo.org/CustomiseTranslations should do this and in theory someone could already use that plugin to customise the opt out translation if they wanted. I think I would very very much like if we referred people that want to customise the text to that plugin as I think < 0.2% or so will ever customise it. 

There's also https://plugins.matomo.org/CustomOptOut which could potentially implement this maybe.

And for those that use it, they might have even quite custom requirements. Eg they track several websites in one Matomo where some are Intranet, some are ... some are ... and they might end up simply needing to use a custom opt out form or something. 

To fix https://github.com/matomo-org/matomo/issues/12810# simply adjusting the opt-out text should do IMO.